test/integration/data/elastic_tensorflow2_main.py

```diff
@@ -32,11 +32,15 @@
                     help='number of batches per commit of the elastic state object')
 parser.add_argument('--epochs', type=int, default=3,
                     help='number of epochs')
+parser.add_argument('--epoch-wait', type=int, default=0,
+                    help='number of seconds each takes')
 parser.add_argument('--logfile', default='/tmp/logfile.txt',
                     help='log file to record results (one line per epoch)')
 parser.add_argument('--discovery-schedule', default='[]',
                     help='JSON string specifying schedule of host updates each epoch')
-parser.add_argument('--exit-schedule',
+parser.add_argument('--discovery-wait', type=int, default=3,
```

Comment:
Nit: please specify units in the help message (e.g., seconds).

Reply:
done: `Time in seconds the worker waits ...`